cache/src/test/kotlin/com/dropbox/android/external/cache4/CacheBuilderTest.kt

```diff
@@ -186,13 +189,13 @@ class CacheBuilderTest {
     @Test
     fun `SystemClock is used when clock was not set explicitly and either expireAfterWrite or expireAfterAccess is positive`() {
```

Comment:
This is my bad: the `expireAfterWrite` and `expireAfterAccess ` actually have no effect on the clock. We can get rid of this test as the default `SystemClock` behavior is covered in the `build cache with defaults` test.